recipes/mailio/all/conanfile.py

```diff
@@ -20,9 +20,9 @@ class mailioConan(ConanFile):
         "fPIC": True,
         "shared": False
     }
-    requires = ["boost/1.75.0", "openssl/1.1.1i"]
-    generators = "cmake"
-    exports_sources = ["CMakeLists.txt", "patches/**"]
+    requires = ["boost/1.75.0", "openssl/1.1.1j"]
+    generators = "cmake", "cmake_find_package"
```

Comment:
You probably don't need cmake generator here and can simplify CMakeLists.txt wrapper as well.
Or even get rid of it.

Reply:
I still need some way to add build dir to CMAKE_MODULE_PATH so find_package can find generated Find<lib>.cmake.

1. One way to it is include cmake generator and call conan_basic_setup().
2. Another way is to include cmake_paths generator and include generated conan_paths.cmake file
 
I searched for cmake_paths line in repo and found none, so I went with the first option.